.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted package name to the issue description
+
+**Packages impacted by this PR:**
+
+
+**Issues associated with this PR:**
+
+
+**Describe the problem that is addressed by this PR:**
+
+
+**What are the possible designs available to address the problem**
```

Comment:
Maybe we can combine this and the one below?

### Alternative designs

> If there is more than one possible solution, describe any alternatives and your reasoning for choosing this approach. Otherwise, delete this section